### ISSUE TITLE
Editor: Improve the inline markdown editor to include buttons to e.g. bold/italic/link etc

### ISSUE THREAD

**Issue by EricWittmann** [Apicurio/apicurio-studio#427]
I'm just using the Ace editor's "markdown" mode as the basis for a simple markdown editor.  It would be easy and useful to have a few buttons available to perform some common tasks e.g. adding a link or making some text bold.

**Comment by tomq42** [comment]
From the point of view of creating actual documentation for an API, having some kind of wysiwyg markdown editor would be awsome. There are a number of javascript libraries that I'm sure provide that.

**Comment by EricWittmann** [comment]
I agree - a wysiwyg editor would be ideal.  The current markdown editor in Apicurio is sort of a hybrid - where you write using markdown syntax and the Ace editor highlights certain elements.  But it's definitely not true wysiwyg.  Hopefully we'll get to the latter eventually - but it's a progression. :)
